### PR TITLE
{data}[intel/2016a] DBD-mysql v4.032

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2584.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/368447a177697e82f418644311500e34 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/7db07570a9385536a8db17e67fd7b3bb for a full test report.

File: easybuild/easyconfigs/d/DBD-mysql/DBD-mysql-4.032-intel-2016a-Perl-5.22.2.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+source_urls = ['http://cpan.metacpan.org/authors/id/C/CA/CAPTTOFU/']
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [
+    ('Perl', '5.22.2'),
+    ('MariaDB', '10.1.13'),
+]
+
+# force static linking of libmysqlclient to dance around problem with unresolved symbols
+configopts = '--libs="-L$EBROOTMARIADB/lib -Wl,-Bstatic -lmysqlclient -Wl,-Bdynamic -lpthread -lz -lrt -lssl -lcrypto"'
```
Comment (wpoely86) [inline easybuild/easyconfigs/d/DBD-mysql/DBD-mysql-4.032-intel-2016a-Perl-5.22.2.eb]: os dep for openssl?

Comment (wpoely86) [inline easybuild/easyconfigs/d/DBD-mysql/DBD-mysql-4.032-intel-2016a-Perl-5.22.2.eb]: add zlib dep

Comment (boegel) [inline easybuild/easyconfigs/d/DBD-mysql/DBD-mysql-4.032-intel-2016a-Perl-5.22.2.eb]: done, thx

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2584.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/674284227381c6e757eebbf8e52de717 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/1954a056e91adba99a275d9c4785089d for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**